cds_ils/importer/providers/cds/rules/multipart.py

```diff
@@ -234,24 +234,18 @@ def volumes_titles(self, key, value):
                 message=" volume title exists but no volume number",
             )
 
-        volume_index = re.findall(r"\d+", val_n) if val_n else None
-        if volume_index and len(volume_index) > 1:
-            raise UnexpectedValue(
-                subfield="n", message=" volume has more than one digit "
-            )
-        else:
-            volume_number = extract_volume_number(
-                val_n, raise_exception=True, subfield="n"
-            )
-            obj = {"title": val_p or volume_title}
-            if val_y:
-                if re.match("\\d+", val_y) and 1600 <= int(val_y) <= 2021:
-                    obj["publication_year"] = int(val_y)
-                else:
-                    raise UnexpectedValue(
-                        subfield="y", message=" unrecognized publication year"
-                    )
-            _insert_volume(_migration, volume_number, obj)
+        volume_number = extract_volume_number(
+            val_n, raise_exception=True, subfield="n"
+        )
+        obj = {"title": val_p or volume_title}
+        if val_y:
+            if re.match("\\d+", val_y) and 1600 <= int(val_y) <= 2021:
```

Comment:
what if we have an older book ?

Reply:
This code was already here, I only removed the restriction for the on digit volumes, I am not sure about this code, maybe we should remove the `1600 <= int(val_y) <= 2021:` constraint? WDYT?